torch/_guards.py

```diff
@@ -618,6 +618,10 @@ def __init__(self, fake_mode):
         # See note [Tensor Fakification and Symbol Caching]
         self.tensor_to_context = WeakTensorKeyDictionary()
 
+        # If this true, Aot Autograd will return output Fake Tensors with appropiate
+        # meta on the first invocation
+        self.fakify_first_call = False
```

Comment:
Happy to put this somewhere else/rename.. Like output_strides, kind of a pain to thread through everything, and hopefully DDP Optimizer is somewhat temporary??

Reply:
haha that's what i was saying last year.  but hopefully for real this time :)

Reply:
Can't think of a better place for it either :)